sensor/kubernetes/listener/resources/deployment_store.go

```diff
@@ -125,14 +126,32 @@ func (ds *DeploymentStore) GetAll() []*storage.Deployment {
 	return ret
 }
 
-// Get returns deployment for supplied id.
-func (ds *DeploymentStore) Get(id string) *storage.Deployment {
+func (ds *DeploymentStore) getWrap(id string) *deploymentWrap {
 	ds.lock.RLock()
 	defer ds.lock.RUnlock()
 
 	wrap := ds.deployments[id]
 	if wrap == nil {
 		return nil
 	}
+	return wrap
```

Comment:
Do we need that `if`? If wrap is `nil` then the returned value will be also `nil`. Or do we care here about typed nils?